anchor/network/src/network.rs

```diff
@@ -74,8 +79,18 @@ impl Network {
     pub async fn run(mut self) {
         loop {
             tokio::select! {
-                _swarm_message = self.swarm.select_next_some() => {
-                    // TODO handle and match swarm messages
+                swarm_message = self.swarm.select_next_some() => {
+                    match swarm_message {
+                        SwarmEvent::Behaviour(behaviour_event) => match behaviour_event {
+                            AnchorBehaviourEvent::Gossipsub(_ge) => {
+                                // TODO handle gossipsub events
+                            },
+                            // TODO handle other behaviour events
+                            _ => todo!()
```

Comment:
nit:
```suggestion
                            _ => {}
```
otherwise this may panic as soon as it runs, as there will be other events. Same applies for the `todo!()` below for the other swarm events

Reply:
Good catch. I had this problem when adding discv5 and I used a log. I applied it here.